livedata/src/main/java/io/getstream/chat/android/livedata/controller/ChannelControllerImpl.kt

```diff
@@ -575,7 +672,7 @@ class ChannelControllerImpl(
         return message
     }
 
-    fun upsertMessages(messages: List<Message>) {
+    private fun upsertMessages(messages: List<Message> = emptyList()) {
```

Comment:
Which case do we need to call to this method without argument and then use an emptyList?
If I am not wrong, this method won't do anything if the `messagesList` is empty. Am I wrong?